### DISCUSSION TITLE
ps2 bios

### DISCUSSION THREAD

**Discussion by nh2122** [archtaurus/RetroPieBIOS#5] [Q&A] [answered]
hello I am looking for the ps2 bios files, specifically 0230A 0230E 0230H and 0230J. If you have them or know where I could find working versions of them it would be greatly appreciated

**Comment by archtaurus** [comment, marked as answer]
You can find them all in the development branch : ps2-0230a-20080220.bin, ps2-0230e-20080220.bin, ps2-0230h-20080220.bin, ps2-0230j-20080220.bin

And I miss "ps2-200ed-20040614.bin". Do you have it?
